cmd/runtimetest/main.go

```diff
@@ -576,6 +576,28 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	t := tap.New()
```

Comment:
This TAP-within-a-TAP approach is a bit strange, but I think it might work…  We can drop this once #308 lands and the `t` instance is getting passed around.

Reply:
Some other modifications can wait until #308 land. Thank you for your advice.